internal/addrs/parse_ref.go

```diff
@@ -328,11 +328,19 @@ func parseRef(traversal hcl.Traversal) (*Reference, tfdiags.Diagnostics) {
 	case "terraform":
 		name, rng, remain, diags := parseSingleAttrRef(traversal)
 		return &Reference{
-			Subject:     TerraformAttr{Name: name},
+			Subject:     NewTerraformAttr("terraform", name),
```

Comment:
We should make "terraform" and "tofu" constants in the addrs package.  Perhaps:
* `addrs.IdentTerraform`
* `addrs.IdentTofu`
or something similar?

Reply:
OK. Extracted them as constant.